prusti-viper/src/encoder/mir/pure/specifications/interface.rs

```diff
@@ -255,6 +258,19 @@ impl<'v, 'tcx: 'v> SpecificationEncoderInterface<'tcx> for crate::encoder::Encod
                 vir_poly::Expr::snap_app(encoded_args[0].clone()),
                 vir_poly::Expr::snap_app(encoded_args[1].clone()),
             )),
+            "prusti_contracts::time_credits" | "prusti_contracts::time_receipts" => {
+                if config::time_reasoning() {
+                    Ok(encode_time_specifications(
+                        self,
+                        &encoded_args[0],
+                        vir_poly::ResourceType::from_function_name(fn_name),
+                        parent_def_id,
+                        span,
+                    ))
+                } else {
+                    Ok(true.into())
```

Comment:
For the time being, trying to use these two functions when the flag is enabled should be an error. If this isn't already detected when parsing specifications, then raise an incorrect specification error here.